.eslintrc

```diff
@@ -20,6 +20,16 @@
     "import/ignore": [".ejs$", ".yml$", ".jsx", "node_modules"]
   },
   "rules": {
-    "lines-between-class-members": "off"
+    "lines-between-class-members": "off",
+    "global-require": "off"
+  },
+  "parser": "@babel/eslint-parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
```

Comment:
Issue: Please pin this `ecma` version. Without doing so, it will lead to lint errors when new rules or syntax becomes available.